pyccl/neutrinos.py

```diff
@@ -15,17 +20,18 @@
 }
 
 
-def Omega_nu_h2(a, *, m_nu, T_CMB=None, T_ncdm=None):
```

Comment:
Doesn't losing this function break api?

Reply:
`Omega_nu_h2` would be a v3 addition, this doesn't currently exist in the API.